plugins/plugin-chart-table/src/utils/DateWithFormatter.ts

```diff
@@ -43,12 +43,11 @@ export default class DateWithFormatter extends Date {
 
     this.input = input;
     this.formatter = formatter;
-  }
-
-  toString(): string {
-    if (this.formatter === String) {
-      return String(this.input);
-    }
-    return this.formatter ? this.formatter(this) : Date.toString.call(this);
```

Comment:
While this code works in production, somehow it doesn't work in the dev mode of Storybook. Moving to instance in constructor to make sure it works in both cases.